build.sbt

```diff
@@ -203,7 +203,12 @@ def commonSettings: Seq[Setting[_]] = eclipseSettings ++ Seq(
 // host and port rather than looking it up from the service locator.
 // See docs/running-in-conductr.md for details.
 def kafkaSettings: Seq[Setting[_]] = Seq(
-  BundleKeys.startCommand += "-Dlagom.broker.kafka.service-name=''"
+  BundleKeys.startCommand ++= Seq(
+    "-Dlagom.broker.kafka.service-name=''",
+    // You may have to edit this list if your Kafka
+    // server is not listening on 127.0.0.1:9092
+    "-Dlagom.broker.kafka.brokers='127.0.0.1:9092'"
+  )
```

Comment:
After merging I realised that I?m not sure this is the recommended approach for production users to override Kafka settings. 
Should we document this as a best practice in the docs for the use case where a developer wants to override the settings, or should we make it obvious here that this is a hack to ease the usage of the `online-auction` sample app?
see https://github.com/lagom/lagom/issues/702